python/trifinger_simulation/visual_objects.py

```diff
@@ -75,40 +76,40 @@ def set_state(self, positions):
             )
 
 
-class CubeMarker:
-    """Visualize a cube."""
-
+class ObjectMarker:
     def __init__(
         self,
-        width,
+        shape_type,
         position,
         orientation,
         color=(0, 1, 0, 0.5),
+        pybullet_client_id=0,
```

Comment:
A small comment on naming: I think it would be less confusing to have the same name for the client id, either `physics_client_id` or `pybullet_client_id` throughout. I was checking the naming in `SimFinger` and `Camera`, seems like `pybullet_client_id` is more common elsewhere, so we could just use this one.

Reply:
Is it currently called `physics_client_id` anywhere?
I was a bit unsure with this.  In pybullet, it is called `physicsClientId`, so using that would be more consistent.  On the otherhand, I think calling it "pybullet" instead of "physics" in our code makes it much easier to understand its meaning.
So I would use `pybullet_client_id` everywhere.

Reply:
Okay, let's keep it `pybullet_client_id` itself then. Right now, it's not called `physics_client_id` anywhere else.